<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="lineWrapMode">
       <enum>QPlainTextEdit::NoWrap</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="actionopen"/>
    <addaction name="actionsave"/>
    <addaction name="actionclose"/>
   </widget>
   <widget class="QMenu" name="menuoption">
    <property name="title">
     <string>option</string>
    </property>
    <widget class="QMenu" name="menufont_size">
     <property name="title">
      <string>font size</string>
     </property>
     <addaction name="action12pt"/>
     <addaction name="action18pt"/>
     <addaction name="action24pt"/>
    </widget>
    <addaction name="menufont_size"/>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menuoption"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionopen">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="text">
    <string>save</string>
   </property>
  </action>
  <action name="actionclose">
   <property name="text">
    <string>close</string>
   </property>
  </action>
  <action name="action12pt">
   <property name="text">
    <string>12pt</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </action>
  <action name="action18pt">
   <property name="text">
    <string>18pt</string>
   </property>
  </action>
  <action name="action24pt">
   <property name="text">
    <string>24pt</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
